### ISSUE TITLE
name 'SynchronizedBatchNorm2d' is not defined

### ISSUE THREAD

**Issue by happy-yu0531** [NVlabs/SPADE#114]
The projects has no models.networks.sync_batchnorm in the document.

**Comment by kei97103** [comment]
please read README.md
you can download sync_batchnorm from https://github.com/vacancy/Synchronized-BatchNorm-PyTorch

**Comment by happy-yu0531** [comment]
I have solved this problem.thanks very much
